base_comment_template/models/res_partner.py

```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    property_comment_template_id = fields.Many2one(
+        comodel_name="base.comment.template",
+        string="Conditions template",
+        company_dependant=True,
```

Comment:
```suggestion
        company_dependent=True,
```
I have already fixed it in v12